SCClassLibrary/Platform/Platform.sc

```diff
@@ -18,12 +18,18 @@ Platform {
 		recordingsDir = this.userAppSupportDir +/+ "Recordings";
 	}
 
+	// cannot redirect 'name' to name because class has a name
```

Comment:
let's implement platformName as an instance member and deprecate name so we can make the interface consistent in the next release